File: Maximum Number of Coins You Can Get.js
var maxCoins = function(piles) {
    
    piles.sort((a,b) => a-b);
    let count=0;
    let output=0;
    let index = piles.length -2
    while(count < piles.length/3){
        output+=piles[index]
        index-=2
        count++
    }
    
    return output;
    
};

File: Maximum Number of Coins You Can Get.py
class Solution:
    def maxCoins(self, piles: List[int]) -> int:
        piles.sort(reverse = True)
        max_coins = 0
        n = 2*(len(piles)//3)
        for i in range(1, n, 2):
            max_coins += piles[i]
        
        return max_coins
